src/frontend/js/widgets/SyllabusCourseRunsList/index.tsx

```diff
@@ -41,7 +43,7 @@ const SyllabusCourseRunsList = ({
   maxArchivedCourseRuns,
 }: {
   courseRuns: CourseRun[];
-  course: CourseLight;
+  course: CourseLight | PacedCourse;
```

Comment:
```suggestion
  course: PacedCourse;
```

With your changes, this component should now accept only `PacedCourse`

Reply:
I've made some changes according to what I've understood.